api/src/main/java/org/apache/apex/api/EmbeddedAppLauncher.java

```diff
@@ -65,6 +66,41 @@ public static EmbeddedAppLauncher newInstance()
   public interface EmbeddedAppHandle extends AppHandle {}
 
   /**
+   * <p>
+   * getDAG.</p>
+   *
+   * @return
+   */
+  public abstract DAG getDAG();
```

Comment:
OK, I see. During  review on github, I missed that EmbeddedAppHandle interface is not changing and thought that methods are added to that interface.